src/contracts/yas_pool.cairo

```diff
@@ -1,23 +1,61 @@
+use yas::numbers::signed_integer::i32::i32;
+use yas::numbers::fixed_point::implementations::impl_64x96::{
+    FP64x96Impl, FP64x96PartialEq, FixedType
+};
+
 #[starknet::interface]
 trait IYASPool<TContractState> {
-    fn initialize(ref self: TContractState);
+    fn initialize(ref self: TContractState, sqrt_price_X96: FixedType);
+}
+
+#[derive(Serde, Copy, Drop, PartialEq, starknet::Store)]
```

Comment:
I don't see the downside of having it either. 
I considered two options, implementing PartialEq separately (since this way it's free) or comparing attribute by attribute in the tests. I believe this approach is cleaner, but I'm open to alternatives.